bentoml/cli/azure_function.py

```diff
@@ -108,18 +114,19 @@ def azure_function():
     )
     @click.option(
         '--function-auth-level',
-        type=click.STRING,
+        type=click.Choice(AZURE_FUNCTION_AUTH_LEVELS),
         default=DEFAULT_FUNCTION_AUTH_LEVEL,
-        help='The authorization level for the deployed Azure function. '
-        'Options are "anonymous", "function", and "admin". Default value is '
-        'function',
+        help=f'The authorization level for the deployed Azure Function. The options '
```

Comment:
Just "The authorization level for the deployed Azure Function." is fine, the --help output already shows all the options for `click.Choice`